src/app/mini-games/sprint/sprint-mini-game.component.ts

```diff
@@ -1,8 +1,221 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy, ViewChild } from '@angular/core';
+
+import { MiniGamesHttpService } from 'src/app/services/mini-games-http.service';
+import { Answers, GameState, Sound, SprintGame, StreakLevel } from 'src/app/core/models/ISprintGame';
+import { getPointsMultiplier, getRandomNumber, getRandomPages, isTranslationCorrect, playSound } from './utils/utils';
+import { MatRipple } from '@angular/material/core';
+import { MatDialog } from '@angular/material/dialog';
+import { Color, DEFAULT_WORDS_DIFFICULTY, MAX_TRAINED_WORDS } from 'src/app/core/constants/sprint-game';
+import { ActivatedRoute, Router } from '@angular/router';
+import { SprintGamePauseExitComponent } from './components/sprint-game-pause-exit/sprint-game-pause-exit.component';
+import { Subscription } from 'rxjs';
 
 @Component({
   selector: 'app-sprint-mini-game',
   templateUrl: './sprint-mini-game.component.html',
   styleUrls: ['./sprint-mini-game.component.scss'],
 })
-export class SprintMiniGameComponent {}
+export class SprintMiniGameComponent implements OnInit, OnDestroy {
+  @ViewChild('answerRipple', { read: MatRipple }) answerRipple!: MatRipple;
+  game: SprintGame = {
+    gameState: GameState.SETUP,
+    words: [],
+    trainedWords: [],
+    trainedWordsByIndexes: [],
+    word: '',
+    wordIndex: 0,
+    wordTranslation: '',
+    isTranslationCorrect: true,
+    streak: 0,
+    points: 0,
+    basePoints: 10,
+    multiplier: 1,
+    isMuted: false,
+    isPaused: false,
+  };
+  loadingWordsMessage = '';
+  isStarterCounterShown = true;
+  closeDialogSubsription?: Subscription;
+  wordsBatch1Subscription?: Subscription;
+  wordsBatch2Subscription?: Subscription;
+  wordsBatch3Subscription?: Subscription;
+
+  constructor(
+    private http: MiniGamesHttpService,
```

Comment:
@Suggestion 
http name is usually used for httpClient. Let's give more readable name for this service for example ```gameService```

Reply:
Changed to gameService